packages/util/src/bytes.ts

```diff
@@ -1,8 +1,33 @@
 import BN from 'bn.js'
-import { intToBuffer, stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
+import { stripHexPrefix, padToEven, isHexString, isHexPrefixed } from 'ethjs-util'
 import { PrefixedHexString, TransformableToArray, TransformableToBuffer } from './types'
 import { assertIsBuffer, assertIsArray, assertIsHexString } from './helpers'
 
+/**
+ * Converts a `Number` into a hex `String`
+ * @param {Number} i
+ * @return {String}
+ */
+export const intToHex = function (i: number) {
+  if (typeof i !== 'number' || isNaN(i) || i === Infinity || i < 0 || i % 1 !== 0) {
+    throw new Error(`Received an invalid integer type: ${i}`)
+  }
+  var hex = i.toString(16); // eslint-disable-line
```

Comment:
should we just do

```
return `0x${i.toString(16)}`
```

(no need to initialize a const?)

or also would this satisfy eslint:

```suggestion
  const hex = i.toString(16)
```